### PR TITLE
cleanup: use err.Wrap when the formatting is not required

### PR DESCRIPTION
**Pull request by Madhu-1**
**Description of your changes:**
Replaced err.Wrapf with err.Wrap when the formatting is not required.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **CommitLint Bot**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
[test ceph]

### CODE REVIEW

File: pkg/operator/ceph/cluster/cluster.go [hunk 1]
```diff
@@ -371,18 +371,18 @@ func (c *cluster) postMonStartupActions() error {
...
 	err = crash.CreateCrashCollectorSecret(c.context, c.Namespace, &c.ownerRef)
 	if err != nil {
-		return errors.Wrapf(err, "failed to create crash collector kubernetes secret")
+		return errors.Wrap(err, "failed to create crash collector kubernetes secret")
 	}
 
 	// Enable Ceph messenger 2 protocol on Nautilus
 	if err := client.EnableMessenger2(c.context, c.Namespace); err != nil {
-		return errors.Wrapf(err, "failed to enable Ceph messenger version 2.")
+		return errors.Wrap(err, "failed to enable Ceph messenger version 2.")
```
Comment (leseb) [inline pkg/operator/ceph/cluster/cluster.go:385]:
```suggestion
		return errors.Wrap(err, "failed to enable Ceph messenger version 2")
```

File: pkg/operator/ceph/cluster/mgr/spec.go [hunk 2]
```diff
@@ -130,14 +130,14 @@ func (c *Cluster) clearHTTPBindFix() error {
 			// the version being upgraded from.
 			if _, err := client.MgrSetConfig(c.context, c.Namespace, daemonID,
 				fmt.Sprintf("mgr/%s/server_addr", module), "", false); err != nil {
-				return errors.Wrapf(err, "failed to set config for an mgr daemon using v2 format.")
+				return errors.Wrap(err, "failed to set config for an mgr daemon using v2 format.")
```
Comment (leseb) [inline pkg/operator/ceph/cluster/mgr/spec.go:133]:
```suggestion
				return errors.Wrap(err, "failed to set config for an mgr daemon using v2 format")
```

File: pkg/operator/ceph/cluster/mgr/spec.go [hunk 3]
```diff
@@ -130,14 +130,14 @@ func (c *Cluster) clearHTTPBindFix() error {
...
 			}
 
 			// this is for the format used in v1.0
 			// https://github.com/rook/rook/commit/11d318fb2f77a6ac9a8f2b9be42c826d3b4a93c3
 			if _, err := client.MgrSetConfig(c.context, c.Namespace, daemonID,
 				fmt.Sprintf("mgr/%s/%s/server_addr", module, daemonID), "", false); err != nil {
-				return errors.Wrapf(err, "failed to set config for an mgr daemon using v1 format.")
+				return errors.Wrap(err, "failed to set config for an mgr daemon using v1 format.")
```
Comment (leseb) [inline pkg/operator/ceph/cluster/mgr/spec.go:140]:
```suggestion
				return errors.Wrap(err, "failed to set config for an mgr daemon using v1 format")
```

File: pkg/operator/ceph/object/user.go [hunk 4]
```diff
@@ -46,12 +46,12 @@ type ObjectUser struct {
...
 	result, err := runAdminCommand(c, "user", "list")
 	if err != nil {
-		return nil, RGWErrorUnknown, errors.Wrapf(err, "failed to list users")
+		return nil, RGWErrorUnknown, errors.Wrap(err, "failed to list users")
 	}
 
 	var s []string
 	if err := json.Unmarshal([]byte(result), &s); err != nil {
-		return nil, RGWErrorParse, errors.Wrapf(err, "failed to read users info result=%s", result)
+		return nil, RGWErrorParse, errors.Wrap(err, "failed to read users info result=%s", result)
```
Comment (leseb) [inline pkg/operator/ceph/object/user.go:54]:
```suggestion
		return nil, RGWErrorParse, errors.Wrapf(err, "failed to read users info result=%s", result)
```

**Comment by Madhu-1** [comment]
@leseb done PTAL

File: pkg/operator/ceph/disruption/clusterdisruption/add.go [hunk 5]
```diff
@@ -42,7 +42,7 @@ func Add(mgr manager.Manager, context *controllerconfig.Context) error {
 	// Add the cephv1 scheme to the manager scheme
 	mgrScheme := mgr.GetScheme()
 	if err := cephv1.AddToScheme(mgr.GetScheme()); err != nil {
-		return errors.Wrapf(err, "failed to add ceph scheme to manager scheme.")
+		return errors.Wrap(err, "failed to add ceph scheme to manager scheme.")
```
Comment (leseb) [inline pkg/operator/ceph/disruption/clusterdisruption/add.go:45]:
```suggestion
		return errors.Wrap(err, "failed to add ceph scheme to manager scheme")
```

File: pkg/operator/ceph/disruption/machinedisruption/add.go [hunk 6]
```diff
@@ -34,10 +34,10 @@ import (
 func Add(mgr manager.Manager, context *controllerconfig.Context) error {
 	mgrScheme := mgr.GetScheme()
 	if err := healthchecking.AddToScheme(mgrScheme); err != nil {
-		return errors.Wrapf(err, "failed to add to healthchecking scheme.")
+		return errors.Wrap(err, "failed to add to healthchecking scheme.")
```
Comment (leseb) [inline pkg/operator/ceph/disruption/machinedisruption/add.go:37]:
```suggestion
		return errors.Wrap(err, "failed to add to healthchecking scheme")
```

File: pkg/operator/ceph/disruption/machinedisruption/add.go [hunk 7]
```diff
@@ -34,10 +34,10 @@ import (
...
 	}
 	if err := cephv1.AddToScheme(mgrScheme); err != nil {
-		return errors.Wrapf(err, "failed to add to ceph scheme.")
+		return errors.Wrap(err, "failed to add to ceph scheme.")
```
Comment (leseb) [inline pkg/operator/ceph/disruption/machinedisruption/add.go:40]:
```suggestion
		return errors.Wrap(err, "failed to add to ceph scheme")
```

File: pkg/operator/ceph/disruption/machinelabel/add.go [hunk 8]
```diff
@@ -42,10 +42,10 @@ const (
 func Add(mgr manager.Manager, context *controllerconfig.Context) error {
 	mgrScheme := mgr.GetScheme()
 	if err := cephv1.AddToScheme(mgrScheme); err != nil {
-		return errors.Wrapf(err, "failed to add scheme to ceph.")
+		return errors.Wrap(err, "failed to add scheme to ceph.")
```
Comment (leseb) [inline pkg/operator/ceph/disruption/machinelabel/add.go:45]:
```suggestion
		return errors.Wrap(err, "failed to add scheme to ceph")
```

File: pkg/operator/ceph/disruption/machinelabel/add.go [hunk 9]
```diff
@@ -42,10 +42,10 @@ const (
...
 	}
 	if err := mapiv1.AddToScheme(mgrScheme); err != nil {
-		return errors.Wrapf(err, "failed to add scheme to mapi.")
+		return errors.Wrap(err, "failed to add scheme to mapi.")
```
Comment (leseb) [inline pkg/operator/ceph/disruption/machinelabel/add.go:48]:
```suggestion
		return errors.Wrap(err, "failed to add scheme to map")
```

File: pkg/operator/ceph/cluster/mon/mon.go [hunk 10]
```diff
@@ -786,16 +786,16 @@ func (c *Cluster) saveMonConfig() error {
 	// Every time the mon config is updated, must also update the global config so that all daemons
 	// have the most updated version if they restart.
 	if err := config.GetStore(c.context, c.Namespace, &c.ownerRef).CreateOrUpdate(c.ClusterInfo); err != nil {
-		return errors.Wrapf(err, "failed to update the global config.")
+		return errors.Wrap(err, "failed to update the global config.")
```
Comment (leseb) [inline pkg/operator/ceph/cluster/mon/mon.go:789]:
```suggestion
		return errors.Wrap(err, "failed to update the global config")
```

**Comment by Madhu-1** [comment]
@leseb done PTAL

**Pull request merged**